web/data/affiliated_pkgs.json

```
@@ -0,0 +1,48 @@
+{
+  "plasmapy": {
+    "maintainer": {
+      "name": null,
+      "contact": null
+    },
+    "urls": {
+      "home": "https://www.plasmapy.org",
+      "repo": "https://github.com/plasmapy/plasmapy",
+      "image": "/images/plasmapy-graphic-circular.png"
+    },
+    "description": {
+      "short": "A open-source Python 3 package for plasma physics.",
+      "long": "PlasmaPy is an open-source community developed Python 3 package for plasma physics.  PlasmaPy intends to be a collection of functionality commonly used and shared between plasma physicists and researchers globally."
+    },
+    "affiliated": true,
```

`RecursionError`? :smiley: 
```suggestion
    "affiliated": false,
```